include/verilated_timing.cpp

```diff
@@ -144,6 +144,66 @@ void VlTriggerScheduler::dump(const char* eventDescription) {
 }
 #endif
 
+//======================================================================
+// VlDynamicTriggerScheduler:: Methods
+
+bool VlDynamicTriggerScheduler::evaluate() {
+#ifdef VL_DEBUG
+    dump();
```

Comment:
```suggestion
    VL_DEBUG_IF(dump(););
```

And have dump always do a dump, generally that's how other dump() routines work.